placement/models.py

```diff
@@ -31,11 +31,7 @@ def get_absolute_url(self):
 
 class application(models.Model):
     name = models.ForeignKey(User, null=True, blank=True, on_delete=models.CASCADE)
-    applied_in = models.ForeignKey(company, null=True, blank=True, on_delete=models.CASCADE)
-    status = models.CharField(default='Applied', max_length=50)
```

Comment:
This status field is required as it will show user application status which is by default Applied as it was removed so while running its giving error 
Exception Type:  IntegrityError
NOT NULL constraint failed: placement_application.status

Reply:
i think that status field is not required because if a object is selected in manytomany field which means that the status for that is accepted.
One of my reason for removing this you haven't changed status anywhere means it is accepted everywhere.
Also if you want i will add that but it will create an extra column. If you want to change the status then changing it in manytomany manager will be good as mentioned in django docs.

Regarding the integrity error you have to migrate the migration file which i have added (if you have any data in this table previously then just delete that before migrating. There are other methods also but i am suggesting the simplest one.)

Reply:
You can also clone my forked branch and use that.